lightning_training_studio/components/sweep.py

```diff
@@ -300,6 +299,19 @@ def _get_objective(self, experiment_id: int):
             )
             setattr(self, f"w_{experiment_id}", objective)
             self.experiments[experiment_id]["stage"] = Stage.PENDING
+
+            # TODO: Remove when display name is merged
+            if hasattr(LightningWork, "_display_name"):
```

Comment:
where do we set `_display_name` as a class attribute of `LightningWork`?

Reply:
Right now,

work.display_name = '...'

If everything works fine, we can add it as an class argument.